pkg/controller/networkpolicy/validate.go

```diff
@@ -710,19 +670,5 @@ func (g *groupValidator) updateValidate(curObj, oldObj interface{}, userInfo aut
 
 // deleteValidate validates the DELETE events of ClusterGroup resources.
 func (g *groupValidator) deleteValidate(oldObj interface{}, userInfo authenticationv1.UserInfo) (string, bool) {
-	oldCG := oldObj.(*crdv1alpha2.ClusterGroup)
```

Comment:
should we also update the admission webhook config and not validate delete for CG?

Reply:
We need this function stub so that groupValidator implements the `validator` interface

Reply:
i don't mean to say remove this stub.. but remove DELETE from webhook config

Reply:
Ah got it. Thanks and updated